lib/improver/tests/ensemble_calibration/ensemble_calibration/test_EnsembleCalibration.py

```diff
@@ -1,510 +0,0 @@
-            self.expected_gaussian_mean_data)
-        self.assertCalibratedVariablesAlmostEqual(
-            calibrated_variance.data,
-            self.expected_gaussian_variance_data)
-
-    @ManageWarnings(
-        ignored_messages=IGNORED_MESSAGES, warning_types=WARNING_TYPES)
-    def test_truncated_gaussian_data_check(self):
-        """
-        Test that the plugin returns an iris.cube.CubeList
-        of wind_speed cubes with the expected data, where the plugin
-        returns a cubelist of, firstly, the predictor and, secondly the
-        variance. The ensemble mean is the predictor.
-        """
-        distribution = "truncated gaussian"
-        plugin = Plugin(distribution)
-        calibrated_predictor, calibrated_variance = plugin.process(
-            self.current_wind_speed_forecast_cube,
-            self.historic_wind_speed_forecast_cube,
-            self.wind_speed_truth_cube)
-        self.assertCalibratedVariablesAlmostEqual(
-            calibrated_predictor.data,
-            self.expected_truncated_gaussian_mean_data)
-        self.assertCalibratedVariablesAlmostEqual(
-            calibrated_variance.data,
-            self.expected_truncated_gaussian_variance_data)
-
-    @ManageWarnings(
-        ignored_messages=IGNORED_MESSAGES, warning_types=WARNING_TYPES)
-    def test_truncated_gaussian_data_check_max_iterations(self):
-        """
-        Test that the plugin returns an iris.cube.CubeList
-        of wind_speed cubes with the expected data, where the plugin
-        returns a cubelist of, firstly, the predictor and, secondly the
-        variance when the maximum number of iterations is specified.
-        The ensemble mean is the predictor.
-        """
-        distribution = "truncated gaussian"
-        plugin = Plugin(distribution,
-                        max_iterations=10000)
-        calibrated_predictor, calibrated_variance = plugin.process(
-            self.current_wind_speed_forecast_cube,
-            self.historic_wind_speed_forecast_cube,
-            self.wind_speed_truth_cube)
-        self.assertCalibratedVariablesAlmostEqual(
-            calibrated_predictor.data,
-            self.expected_truncated_gaussian_mean_data)
-        self.assertCalibratedVariablesAlmostEqual(
-            calibrated_variance.data,
-            self.expected_truncated_gaussian_variance_data)
-
-
-@unittest.skipIf(
-    STATSMODELS_FOUND is False, "statsmodels module not available.")
-class Test_process_with_statsmodels(SetupCubes, SetupExpectedResults,
-                                    EnsembleCalibrationAssertions):
-
-    """Additional tests for the process method when the statsmodels module
-    is available. The statsmodels module is used for creating an initial
-    guess for the coefficients that will be solved as part of the
-    calibration."""
-
-    @ManageWarnings(
-        ignored_messages=IGNORED_MESSAGES, warning_types=WARNING_TYPES)
-    def test_gaussian_realizations_data_check(self):
```

Comment:
The lost bit of the test is explaining something that was found in the long hours Gavin spent looking at this, so it might be nice to retain it somewhere if possible. I'll defer to the opinion of @gavinevans.